bootstrap4/.js/src/main/scala/io/udash/bootstrap/utils/UdashIcons.scala

```diff
@@ -1404,6 +1598,7 @@ object UdashIcons {
       def wrench: CssStyle = fas("wrench")
       def xRay: CssStyle = fas("x-ray")
       def yenSign: CssStyle = fas("yen-sign")
+      def yinYang: CssStyle = fas("yin-yang")
     }
```

Comment:
Do you think we could add a test for rendering these based on e.g. second example `https://developer.mozilla.org/en-US/docs/Web/API/FontFaceSet/check#Examples`?

Reply:
According to the docs, the two-arg variant of this method

> does not check for individual glyph coverage

([explanation](http://lists.w3.org/Archives/Public/www-style/2015Aug/0330.html)). However I've finally managed to implement a test for FontAwesome API/statics compatibility, please check it out under `UdashIconsTest`.